lib/worker.ex

```diff
@@ -3,43 +3,60 @@ defmodule Monarch.Worker do
   Defines the Monarch Worker for running Oban jobs.
   """
 
-  use Oban.Worker
+  use Oban.Worker,
+    unique: [
+      fields: [:args, :worker],
+      keys: [:job, :repo]
+    ]
```

Comment:
Looks like every single pod tried enqueuing the same job on boot this morning.

This fixes that; oban will silently ignore any job that is already running.